### PR TITLE
Ayr 441/start screen

### PR DESCRIPTION
**Pull request by zaiziclebby**
## Changes in this PR

-  Created start-page.html
- Created start_page route
- Created start-page.css, and added styles 
- Linked css page to base.html
- Created how-to-use-this-service.html page (just contains placeholder text for now) 
- Created how_to_use route 
- Linked how-to-use-this-service.html page to start-page 
- Configured pre-commit hook to work with Poetry

## JIRA ticket

https://national-archives.atlassian.net/browse/AYR-441

### After
Screenshot 2023-11-01 at 11 06 25
Screenshot 2023-11-01 at 11 11 48

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/templates/main/how-to-use-this-service.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+
+{% block pageTitle %}Dashboard – {{config['SERVICE_NAME']}} – GOV.UK{% endblock %}
```
Comment (colinbowen) [inline app/templates/main/how-to-use-this-service.html]: Do we want to have Dashboard as title for both start page & how to use this service?

File: app/templates/main/how-to-use-this-service.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
```
Comment (colinbowen) [inline app/templates/main/how-to-use-this-service.html]: This import isn't being used, could we remove it?

File: app/templates/start-page.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
```
Comment (colinbowen) [inline app/templates/start-page.html]: Same with this import

**Comment by colinbowen** [comment]
Hi Candace, just a couple points which we should take a look at. You'll need to pull / rebase to sort out the conflict in routes.py. There's also differing spacing between the BETA banner & the corresponding title for each of the pages.

File: app/templates/main/how-to-use-this-service.html (hunk 1 above)
Comment (zaiziclebby) [reply app/templates/main/how-to-use-this-service.html:4]: removed from how to use this service

File: app/templates/main/how-to-use-this-service.html (hunk 2 above)
Comment (zaiziclebby) [reply app/templates/main/how-to-use-this-service.html]: done

File: app/templates/start-page.html (hunk 3 above)
Comment (zaiziclebby) [reply app/templates/start-page.html]: done

File: app/templates/main/how-to-use-this-service.html
Comment (colinbowen) [inline app/templates/main/how-to-use-this-service.html:1]: I think the title may have been removed by mistake:

```
{% block pageTitle %}Dashboard – {{config['SERVICE_NAME']}} – GOV.UK{% endblock %}

{% block beforeContent %}
  {{ super() }}

{% endblock %}
```

All pages will need to eventually have a page title.

Comment (colinbowen) [reply app/templates/main/how-to-use-this-service.html:1]: Just a generic one should do for now. Maybe @Terry-Price has some thoughts as to what the naming should be for page titles?

File: app/templates/main/sign-out.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+
+{% block pageTitle %}Dashboard – {{config['SERVICE_NAME']}} – GOV.UK{% endblock %}
```
Comment (colinbowen) [inline app/templates/main/sign-out.html]: This needs updating.

**Comment by colinbowen** [comment]
Looks good, we'll need to get the flask app connected to keycloak again after the user clicks on start now.

**Review by colinbowen — APPROVED** [review]

**Pull request closed without merging**